server/controllers/refer/courseParticipationsController.test.ts

```diff
@@ -42,11 +42,9 @@ describe('CourseParticipationsController', () => {
     request = createMock<Request>({ params: { referralId }, user: { token } })
     response = Helpers.createMockResponseWithCaseloads()
     courseParticipationsController = new CourseParticipationsController(courseService, personService, referralService)
-    ;(CourseParticipationUtils.summaryListOptions as jest.Mock).mockImplementation(
-      (_courseParticipationWithName, _referralId, _withActions = { change: true, remove: true }) => {
-        return summaryListOptions
-      },
-    )
+    courseService.presentCourseParticipation.mockResolvedValue(summaryListOptions)
+    courseParticipationsController = new CourseParticipationsController(courseService, personService, referralService)
```

Comment:
You're assigning the controller variable twice here

Reply:
oops, don't know how that slipped in